src/blur-hash/ci.service.js

```diff
@@ -52,16 +52,20 @@ export default class CIResponsive {
     }
   }
 
-  process(isUpdate) {
+  process(isUpdate, rootElement = document) {
     let images, backgroundImages;
     const windowScreenBecomesBigger = this.innerWidth < window.innerWidth;
 
+    if (rootElement !== document && !rootElement instanceof HTMLElement) {
```

Comment:
probably, you wanted to do

rootElement !== document && !(rootElement instanceof HTMLElement)

@Nargonath , right?

Reply:
Yes indeed, good catch @dzmitry-stramavus . My bad I thought `instanceof` had a higher precedence but you're right (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence).